src/gas/fetchGasEstimatesViaEthFeeHistory/calculateBaseFeeRange.ts

```diff
@@ -0,0 +1,22 @@
+import { fromWei } from 'ethjs-unit';
```

Comment:
As before, you can check against https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/master/src/eip1559/getInfuraGasPrices.ts and https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/master/src/eip1559/feeHistory.ts for the original implementation of this logic.